sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -262,8 +262,14 @@ class SQLContext(@transient val sparkContext: SparkContext)
    *
    * @group userf
    */
-  def parquetFile(path: String): DataFrame =
-    DataFrame(this, parquet.ParquetRelation(path, Some(sparkContext.hadoopConfiguration), this))
+  @scala.annotation.varargs
+  def parquetFile(paths: String*): DataFrame =
```

Comment:
as commented on the other pr, use
```scala
def parquetFile(path: String, paths: String*): DataFrame
```

to make sure this is not ambiguous if we overload the function with another varargs